npm-packages/meteor-installer/install.js

```diff
@@ -199,8 +199,15 @@ function download() {
     }
 
     if (isWindows()) {
-      decompress();
-      return;
+      const isWindows10OrLater = fs.existsSync(path.resolve('C:/Windows/System32', 'tar.exe'));
+      if (isWindows10OrLater) {
```

Comment:
Is this actually a valid Windows 10 check? Or can we just call it "hasNativeTar" or something similar? I'm not a Windows expert but I guess Windows 11 has it as well.

Reply:
Yes, it is a valid check for all editions of Windows starting from [Windows 10 Insider Preview Build 17063](https://www.thomasmaurer.ch/2017/12/tar-and-curl-on-windows-10/).

In other words, all copies of Windows sold since 2018, starting from [Windows 10 version 1803](https://bsmadhu.wordpress.com/2018/08/23/curltar-ssh-tools-on-windows-10/), now ship with `tar.exe` in `C:\Windows\System32`. Of course this includes Windows 11 which was released in 2021.

I've renamed `isWindows10OrLater` to `hasNativeTar` as suggested.